doc-templates/SiriAzureUpdater.md

```diff
@@ -0,0 +1,33 @@
+# Siri Azure Updater
```

Comment:
I know this is a mess but can you put this in docs-templates/sandbox/?

Reply:
Fixed